src/Controller/ContentController.php

```diff
@@ -24,20 +24,28 @@
 use Pimcore\Model\DataObject\Manufacturer;
 use Pimcore\Model\DataObject\Service;
 use Pimcore\Translation\Translator;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Entity;
```

Comment:
```suggestion

```

Reply:
replace `use Pimcore\Controller\Configuration\ResponseHeader;` on line 22 to `use Pimcore\Controller\Attribute\ResponseHeader;`